src/tests/Microsoft.Agents.BotBuilder.Tests/ActivityHandlerTests.cs

```diff
@@ -749,6 +760,115 @@ public async Task TestGetSearchInvokeValue_MissingQueryTextThrows()
             await AssertErrorThroughInvokeAdapter(activity, "Missing queryText property for search");
         }
 
+        [Fact]
+        public async Task OnTurnAsync_ShouldThrowArgumentException()
```

Comment:
This is the style we use for test names:
```suggestion
        public async Task OnTurnAsync_ShouldThrowOnNullContext()
```

Reply:
renamed